ignite/services/network/publish.go

```diff
@@ -221,3 +222,42 @@ func (n Network) Publish(ctx context.Context, c Chain, options ...PublishOption)
 	}
 	return createChainRes.LaunchID, campaignID, mainnetID, nil
 }
+
+func (n Network) SendFromAccountRequest(launchID uint64, amount sdk.Coins) error {
```

Comment:
```suggestion
func (n Network) SendAccountRequestForCoordinator(launchID uint64, amount sdk.Coins) error {
```

Maybe something like this is more obvious?